### PR TITLE
Improvement: Disable multi-select models of different toolpath types

### PR DESCRIPTION
**Pull request by meiliang1123**

### CODE REVIEW

**Comment by meiliang1123** [comment]
LU-709

File: src/app/containers/App.jsx [hunk 1]
```diff
@@ -31,7 +32,8 @@ import Cnc from './Cnc';
 import Settings from './Settings';
 import CaseLibrary from './CaseLibrary';
 import styles from './App.styl';
-// import recoverEnvironmentModal from '../modals/modal-recover-environment';
+import '../styles/react-toastify.styl';
```
Comment (parachvte) [inline src/app/containers/App.jsx:35]: Checkout `src/app/styles/vendor.styl`, this is how we add vendor style.

File: src/app/containers/App.jsx [hunk 2]
```diff
@@ -377,6 +379,18 @@ class App extends PureComponent {
                         )}
                     </div>
                 </div>
+                {/* https://fkhadra.github.io/react-toastify/api/toast-container */}
```
Comment (parachvte) [inline src/app/containers/App.jsx:382]: Doc about 3rd party React Component is not needed (and not recommended cuz no information for later use).

Recommended way to do this is to wrapper 3rd party component, and then add docs on wrapped component. :)

**Review by parachvte — CHANGES_REQUESTED** [review]
- style import changes required
- component wrapper recommended
- improvement approved

**Pull request merged**